apps/axistream/src/hls_macros.hh

```diff
@@ -0,0 +1,8 @@
+#pragma once
+
```

Comment:
Worth a comment?  I assume this is to avoid undefined pragma warnings?   Personally, I just pass the corresponding gcc flag.

Reply:
Yes, to avoid the undefined pragma warnings. The gcc flag would also work.